script/train-events.lua

```diff
@@ -121,6 +122,12 @@ function TrainArrives(train)
 
         setLamp(stop, "blue", 1)
 
+        end
+        -- if train had a delivery fire completed event
+        log("Train arrives at depot: "..train.id.." : "..serpent.block(delivery))

```

Comment:
Never leave debug logging in pull requests
LTN has a debug switch to use if it should remain in releases.